arcsi/__init__.py

```diff
@@ -64,4 +65,16 @@ def create_app(config_file):
     app.register_blueprint(api.arcsi)
     app.register_blueprint(view.router)
 
+    ### swagger specific ###
+    SWAGGER_URL = '/docs'
```

Comment:
I know this is a nitpick but can you please modify it from `docs` to `doc` ? We try to use an API endpoint naming schema where everything is singular

Reply:
Fixed it 👍